crawl-ref/source/makeitem.cc

```diff
@@ -640,7 +638,6 @@ static void _generate_missile_item(item_def& item, int force_type,
             random_choose_weighted(50, MI_STONE,
                                    20, MI_ARROW,
                                    12, MI_BOLT,
-                                   12, MI_SLING_BULLET,
                                    10, MI_DART,
```

Comment:
Please give this 12 weight to stone generation (so that stones get total 62 weight), rather than removing it altogether. Removing it altogether causes the weight to effectively be distributed evenly across all ranged weapon types, which will cause more throwing weapons to generate than expected.